content/en/security_platform/application_security/getting_started/dotnet.md

```diff
@@ -0,0 +1,174 @@
+---
+title: .NET Applications
+kind: documentation
+code_lang: dotnet
+type: multi-code-lang
+code_lang_weight: 10
+further_reading:
+    - link: 'https://github.com/DataDog/dd-trace-dotnet'
+      tag: 'GitHub'
+      text: '.NET Datadog Library source code'
+---
+
+{{% appsec-getstarted %}}
+
+## Get started
+
+1. **Download the [Datadog .NET library][1]**, at least version 2.2.0, for your target operating system architecture.
+
+   For information about which language and framework versions are supported by the library, see [Compatibility][2].
+
+2. **Enable Application Security Monitoring** by setting the `DD_APPSEC_ENABLED` environment variable to `true`. For example, on Windows self-hosted, run the following PowerShell snippet as part of your application start up script:
+   ```
+   $target=[System.EnvironmentVariableTarget]::Process
+   [System.Environment]::SetEnvironmentVariable("DD_APPSEC_ENABLED","true",$target)
+   ```
+
+   **Or** one of the following methods, depending on where your application runs:
+
+   {{< tabs >}}
+{{% tab "Windows self-hosted" %}}
+
+In a Windows console: 
+
+```
+rem Set environment variables
+SET CORECLR_ENABLE_PROFILING=1
+SET CORECLR_PROFILER={846F5F1C-F9AE-4B07-969E-05C26BC060D8}
+SET DD_APPSEC_ENABLED=true
+
+rem Start application
+dotnet.exe example.dll
+```
+
+{{% /tab %}}
+{{% tab "IIS" %}}
+
+Run the following PowerShell command as administrator to configure the necessary environment variables in the registry `HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Control\Session Manager\Environment` and restart IIS.
+```
+$target=[System.EnvironmentVariableTarget]::Machine
+[System.Environment]::SetEnvironmentVariable("DD_APPSEC_ENABLED","true",$target)
+net stop was /y
+net start w3svc
+```
+
+**Or**, for IIS services exclusively, on WAS and W3SVC with Powershell as an administrator, run:
+
+```
+$appsecPart = "DD_APPSEC_ENABLED=true"
+[string[]] $defaultvariable = @("CORECLR_ENABLE_PROFILING=1", "CORECLR_PROFILER={846F5F1C-F9AE-4B07-969E-05C26BC060D8}", $appsecPart)
+
+function Add-AppSec {
+
+    param (
+        $path
+    )
+    $v = (Get-ItemProperty -Path $path).Environment
+    If ($v -eq $null) {
+        Set-ItemProperty -Path $path -Name "Environment" -Value $defaultvariable
+    }
+    ElseIf (-not ($v -match $appsecPart)) {
+        $v += " " + $appsecPart;
+        Set-ItemProperty -Path $path -Name "Environment" -Value $v
+    }
+}
+Add-AppSec -path "HKLM:SYSTEM\CurrentControlSet\Services\WAS\"
+Add-AppSec -path "HKLM:SYSTEM\CurrentControlSet\Services\W3SVC\"
+
+net stop was /y
+net start w3svc
+```
+
```

Comment:
```suggestion
**Or**, to avoid editing registry keys, edit the application settings in the `web.config` file of your application: 

<configuration>
  <appSettings>
        <add key="DD_APPSEC_ENABLED" value="true"/>
  </appSettings>
</configuration>
```

Reply:
```suggestion
**Or**, to avoid editing registry keys, edit the application settings in the `web.config` file of your application: 
```
	<configuration>
	  <appSettings>
	        <add key="DD_APPSEC_ENABLED" value="true"/>
	  </appSettings>
	</configuration>
```
```